great_expectations/render/types/__init__.py

```diff
@@ -501,18 +500,20 @@ def to_json_dict(self):
 class RenderedAtomicValue(DictDot):
     def __init__(
         self,
+        schema: Optional[dict] = None,
         template: Optional[str] = None,
         params: Optional[dict] = None,
-        schema: Optional[dict] = None,
         header: Optional["RenderedAtomicValue"] = None,
         header_row: Optional[List["RenderedAtomicValue"]] = None,
         table: Optional[List[List["RenderedAtomicValue"]]] = None,
         graph: Optional[dict] = None,
+        kwargs: Optional[dict] = None,
     ) -> None:
+        self.schema: Optional[dict] = schema
```

Comment:
`self.schema` is used for other types so I moved it out from under `StringValueType`.